### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by shuoer86**

### CODE REVIEW

**Review by macfarla — APPROVED** [review]
Thanks! Approved pending DCO fix. @shuoer86 you need to include signoff in all your commits. Please follow the instructions to fix

**Comment by macfarla** [comment]
> Thanks! Approved pending DCO fix. @shuoer86 you need to include signoff in all your commits. Please follow the instructions to fix https://github.com/hyperledger/besu/pull/6036/checks?check_run_id=17737846739

and also run `gradlew spotlessApply`

**Comment by shuoer86** [comment]
@macfarla Thanks for your review. I have done DCO.

**Pull request merged**
